tests/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.SourceGenerators.Testing.UnitTests/SourceGeneratorValidationTests.cs

```diff
@@ -36,6 +36,42 @@ public async Task AddSimpleFile()
             }.RunAsync();
         }
 
+        [Fact]
+        public async Task MultipleFilesAllowEitherOrder()
```

Comment:
This test (or another) seems like it would be more interesting if the generated files didn't have exactly the same content, and empty besides.